source/game.room.base.bmx

```diff
@@ -436,18 +436,42 @@ Type TRoomBase extends TOwnedGameObject {_exposeToLua="selected"}
 	'easy accessor to block a room using predefined values
 	Method SetBlockedState:int(blockedState:int = 0)
 		local time:int = 0
+		local renovationBaseCost:int = 0
 
 		'=== BOMB ===
 		if blockedState & BLOCKEDSTATE_BOMB > 0
 			'"placerholder rooms" (might get rent later)
 			if owner = 0 and IsUsableAsStudio()
-				time = 240 * TWorldTime.MINUTELENGTH
+				time = randRange(6,12) * 30 * TWorldTime.MINUTELENGTH
 			'rooms like movie agency
 			elseIf owner = 0
-				time = 90 * TWorldTime.MINUTELENGTH
+				time = randRange(3,6) * 30 * TWorldTime.MINUTELENGTH
 			'player rooms
 			elseIf owner > 0
-				time = 60 * TWorldTime.MINUTELENGTH
+				Local difficulty:TPlayerDifficulty = GetPlayerDifficulty(owner)
+				time = randRange(3,6) * 15 * TWorldTime.MINUTELENGTH
+				Select difficulty.GetGUID()
+					'TODO difficulty.renovationTimeMod
+					'TODO difficulty.renovationBaseCost
+					Case "easy"
+						time:* 0.75
+						renovationBaseCost = 25000
+					Case "normal"
+						renovationBaseCost = 50000
+					Case "hard"
+						time:* 1.5
+						renovationBaseCost = 75000
+				EndSelect
+				Select getNameRaw()
```

Comment:
`getNameRaw()` gibt nur bei _originalen_ Studios "studio" zurueck. Bei "umfunktionierten" Raeumen also deren Namen. Hier waere wohl `getName()` geeigneter.

Reply:
Studios habe ich mit dem IsUsedAsStudio unmittelbar darüber behandelt.

Reply:
Mein Kommentar war zu einem der Commits, in dem noch "studio" als case-Eintrag stand. Was halt nur fuer original-Studios so ist.

![image](https://user-images.githubusercontent.com/2625226/205598013-2a311311-54a6-48c6-af4e-372e4ffcc1f2.png)